models/articles_model.js

```diff
@@ -1,18 +1,16 @@
 const db = require("../db/connection");
 
 exports.selectArticlesById = async (article_id) => {
-  if (isNaN(Number(article_id))) {
-    return Promise.reject({ status: 400, msg: "Bad Request" });
+  try {
+    const result = await db.query(
+      "SELECT * FROM articles WHERE article_id = $1;",
+      [article_id]
+    );
+    if (result.rows.length === 0) {
+      return Promise.reject({ status: 404, msg: "Not Found" });
+    }
+    return result.rows[0];
+  } catch (err) {
+    return Promise.reject(err);
```

Comment:
You will need to add some logic here to pass the test that you will need to refactor. Please do hit the help button if you are stuck with it 🙂